### PR TITLE
staticjinja: 3.0.1 -> 4.1.0, pythonPackages.docopt-ng: init at 0.7.2

### PR DESCRIPTION
**Pull request by fgaz**
###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [x] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [21.11 Release Notes (or backporting 21.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2111-release-notes)
  - [x] (Package updates) Added a release notes entry if the change is major or breaking
  - [x] (Module updates) Added a release notes entry if the change is significant
  - [x] (Module addition) Added a release notes entry if adding a new NixOS module
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by r-rmcgibbo** [comment]
Result of `nixpkgs-review pr 133635` at ff988372 run on aarch64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>5 packages built successfully:</summary>
  <ul>
    <li>nixos-install-tools</li>
    <li>python38Packages.docopt-ng</li>
    <li>python38Packages.staticjinja</li>
    <li>python39Packages.docopt-ng</li>
    <li>staticjinja (python39Packages.staticjinja)</li>
  </ul>
</details>
<details>
  <summary>1 suggestion:</summary>
  <ul>
    <li>warning: <a href="https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/maintainers-missing.md">maintainers-missing</a>

Package does not have a maintainer. Consider adding yourself?

Near pkgs/development/python-modules/docopt-ng/default.nix:18:3:
```
   |
18 |   meta = with lib; {
   |   ^
```
</li>
  </ul>
</details>

File: pkgs/development/python-modules/docopt-ng/default.nix [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    inherit pname version;
+    sha256 = "sha256-hs7qAy8M+lnmB3brDPOKxzZTWBAihyMg9H3IdGeNckQ=";
+  };
+
+  pythonImportsCheck = [ "docopt" ];
+  doCheck = false; # no tests in the package
+
+  meta = with lib; {
+    description = "More-magic command line arguments parser. Now with more maintenance!";
+    homepage = "https://github.com/bazaar-projects/docopt-ng";
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/docopt-ng/default.nix:20]: Missing maintainer.

Comment (fgaz) [reply pkgs/development/python-modules/docopt-ng/default.nix:20]: > `Now with more maintenance!`

ironic :)

done

**Comment by fgaz** [comment]
And fixed the merge conflict

**Comment by SuperSandro2000** [comment]
This is a semi-automatic executed nixpkgs-review with [nixpkgs-review-checks extension](https://github.com/SuperSandro2000/nixpkgs-review-checks). It is checked by a human on a best effort basis and does not build all packages (e.g. lumo, tensorflow or pytorch).

Result of `nixpkgs-review pr 133635` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
<summary>
1 package blacklisted:
</summary>
<ul>
<li>tests.nixos-functions.nixos-test</li>
</ul>
</details>
<details>

  <summary>1 package failed to build and are new build failure:</summary>
  <ul>
<li>nixos-install-tools: log was empty</li>
  </ul>
</details>
<details>
<summary>
4 packages built:
</summary>
<ul>
<li>python38Packages.docopt-ng</li>
<li>python38Packages.staticjinja</li>
<li>python39Packages.docopt-ng</li>
<li>staticjinja (python39Packages.staticjinja)</li>
</ul>
</details>

**Pull request merged**